sandbox.go

```diff
@@ -536,6 +537,11 @@ func (sb *sandbox) resolveName(req string, networkName string, epList []*endpoin
 }
 
 func (sb *sandbox) SetKey(basePath string) error {
+	start := time.Now().UnixNano()
+	defer func() {
+		log.Debugf("sandbox set key processing took %v ms for container %s", (time.Now().UnixNano()-start)/1e6, sb.ContainerID())
```

Comment:
Division is expensive. Do this instead
```
start := time.Now()
defer func() {
    log.Debugf("sandbox set key processing took %s for container %s", time.Now().Sub(start), containerID)
}
```

If you you pass the `time.Duration` type it will nicely print in ms granularity. Also you didn't pass the container ID in your PR.